manifests/overlays/additional/jupyter-trustyai-cpu-py312-ubi9-imagestream.yaml

```diff
@@ -25,17 +25,17 @@ spec:
           [
             {"name": "JupyterLab","version": "4.4"},
             {"name": "TrustyAI", "version": "0.6"},
-            {"name": "Transformers", "version": "4.49"},
+            {"name": "Transformers", "version": "4.52"},
             {"name": "Datasets", "version": "3.4"},
             {"name": "Accelerate", "version": "1.5"},
             {"name": "Torch", "version": "2.6"},
             {"name": "Boto3", "version": "1.37"},
             {"name": "Kafka-Python-ng", "version": "2.2"},
             {"name": "Kfp", "version": "2.12"},
-            {"name": "Matplotlib", "version": "3.6"},
-            {"name": "Numpy", "version": "1.24"},
+            {"name": "Matplotlib", "version": "3.10"},
+            {"name": "Numpy", "version": "1.26"},
             {"name": "Pandas", "version": "1.5"},
-            {"name": "Scikit-learn", "version": "1.5"},
+            {"name": "Scikit-learn", "version": "1.7"},
             {"name": "Scipy", "version": "1.15"},
             {"name": "Odh-Elyra", "version": "4.2"},
             {"name": "PyMongo", "version": "4.11"},
```

Comment:
@coderabbitai please check Sklearn-onnx version in the relevant Pipfile/pipfile.lock/requirements.txt